vignettes/how_to_transform.Rmd

```diff
@@ -0,0 +1,387 @@
+---
+title: "How to transform variables"
+output: rmarkdown::html_vignette
+vignette: >
+  %\VignetteIndexEntry{How to guides}     
+  %\VignetteEngine{knitr::rmarkdown}  %\VignetteEncoding{UTF-8}
+---
+
+```{r setup, include=FALSE}
+knitr::opts_chunk$set(echo = TRUE, collapse = TRUE, comment = "#>")
+source("../R/test_data_generator.R")
+```
+
+# How to transfrom variables
```

Comment:
@Rhan43 Hmmm. Reading this section gave me the impression that we should use the term recode as opposed to transform. Perhaps not a big issue and keep as is. For many people, transform implies a broader range of calculations such as normalizing, centering, etc. We are really focusing on a specific type of transformation that people more often call recoding.

Reply:
@DougManuel, I also had this debate as I created the documentation. I agree that recode is the better term. I've changed it throughout the documentation